README.md

```diff
@@ -13,8 +13,9 @@
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/byjg/php-swagger-test/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/byjg/php-swagger-test/?branch=master)
 
 
-A set of tools for test your REST calls based on the OpenApi specification using PHPUnit. Currently this library supports 
-the specifications `2.0` and `3.0`.
+A set of tools for test your REST calls based on the OpenApi specification using PHPUnit. 
+Currently, this library supports the OpenApi specifications `2.0` (former swagger) and `3.0`.
+Some features like callbacks and links weren't implemented. 
```

Comment:
I think the referencing to components in external JSON files is missing in this list, like described in "[4.7.23 Reference Object](http://spec.openapis.org/oas/v3.0.3#reference-object)" of the specification.
Or did I miss something and the feature is there?

Reply:
You are right. The features  `4.7.23.3` and `4.7.23.4` are not implemented. It is an interesting missing feature.

Reply:
For now, I am documenting that this feature is missing. But in some moment I am planning to implement this :)